apps/ledger-live-desktop/src/newArch/features/AnalyticsOptInPrompt/hooks/useCommonLogic.tsx

```diff
@@ -61,13 +61,8 @@ export const useAnalyticsOptInPrompt = ({ entryPoint }: Props) => {
     () =>
       isEntryPointIncludedInFlagParams &&
       lldAnalyticsOptInPromptFlag?.enabled &&
-      (!hasSeenAnalyticsOptInPrompt || entryPoint === EntryPoint.onboarding),
-    [
-      lldAnalyticsOptInPromptFlag,
-      hasSeenAnalyticsOptInPrompt,
-      entryPoint,
-      isEntryPointIncludedInFlagParams,
-    ],
+      !hasSeenAnalyticsOptInPrompt,
+    [lldAnalyticsOptInPromptFlag, hasSeenAnalyticsOptInPrompt, isEntryPointIncludedInFlagParams],
```

Comment:
@LucasWerey I assumed the `entryPoint === EntryPoint.onboarding` was there to help with the development (so that when clicking on reset onboarding the prompt shows again). Is it correct or was there a different reason for this ?